content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -265,6 +262,9 @@ For a reference to old feature gates that are removed, please refer to
 | `DisableAcceleratorUsageMetrics` | `false` | Alpha | 1.19 | 1.19 |
 | `DisableAcceleratorUsageMetrics` | `true` | Beta | 1.20 | 1.24 |
 | `DisableAcceleratorUsageMetrics` | `true` | GA | 1.25 |- |
+| `DownwardAPIHugePages` | `false` | Alpha | 1.20 | 1.20 |
+| `DownwardAPIHugePages` | `false` | Beta | 1.21 | 1.26 |
```

Comment:
Is this accurate?
My understanding is that the feature was labelled Beta in 1.21, but still defaulted to False.
It had been defaulted to True since 1.22, as a Beta feature.

Reply:
Ah yes we need 4 entries here.